tests/conftest.py

```diff
@@ -63,13 +57,12 @@ def log_has_re(line, logs):
 
 def get_args(args):
     return Arguments(args).get_parsed_arg()
+# Source: https://stackoverflow.com/questions/29881236/how-to-mock-asyncio-coroutines
 
 
-# Source: https://stackoverflow.com/questions/29881236/how-to-mock-asyncio-coroutines
 def get_mock_coro(return_value):
```

Comment:
I'd ... keep this comment where it was - now it seems to belong to `get_args` - but it's really for `mock_coro` ...